src/openforms/utils/admin.py

```diff
@@ -1,9 +1,30 @@
+from django.conf import settings
+from django.contrib import admin
 from django.utils.translation import gettext_lazy as _
 
+from cookie_consent.admin import LogItemAdmin as CookieLogAdmin
+from cookie_consent.models import LogItem as CookieLog
 from djchoices import ChoiceItem, DjangoChoices
 
 
 class SubmitActions(DjangoChoices):
     save = ChoiceItem("_save", _("Save"))
     add_another = ChoiceItem("_addanother", _("Save and add another"))
     edit_again = ChoiceItem("_continue", _("Save and continue editing"))
+
+
+# defaults to True
+if getattr(settings, "COOKIE_CONSENT_LOG_ENABLED", True):
```

Comment:
does this properly work with tests and `@override_settings`? Generally setting evaluation (since it's a LazyObject) at the module level is a bad idea